src/functions.js

```diff
@@ -166,6 +166,14 @@ defineFunction("\\color", {
     };
 });
 
+// \color is handled in Parser.js's parseImplicitGroup
```

Comment:
Yes.  @gagern and I had discussed this at some point, and it felt like *all* the implicit group parsing could be moved to `functions.js`.  Worth considering in the future...